src/main/java/tools/jackson/databind/deser/impl/UnwrappedPropertyHandler.java

```diff
@@ -90,7 +90,7 @@ public PropertyValueBuffer processUnwrappedCreatorProperties(JsonParser original
     }
 
     @SuppressWarnings("resource")
-    public Object processUnwrapped(JsonParser originalParser, DeserializationContext ctxt,
+    public Object processUnwrapped(DeserializationContext ctxt,
```

Comment:
NO, you cannot just go randomly remove a method signature of publicly exposed API that would break all extensions.

```suggestion
    public Object processUnwrapped(JsonParser originalParser, DeserializationContext ctxt,
```

Reply:
Definitely no to breaking internal API change -- would have been ok for 3.0.0, but not within patch for sure, and since it's harmless not for minor version update either.

Reply:
@JooHyukKim @cowtowncoder 

Thank you for your explanation :)